### ISSUE TITLE
Instant preview showing Green Screen

### ISSUE THREAD

**Issue by knightcube** [google-ar/arcore-unity-sdk#643]
SPECIFIC ISSUE ENCOUNTERED
I get this green screen when I try to run instant preview in 
![Screenshot_20190918-140843](https://user-images.githubusercontent.com/22871961/65132945-70a1bd00-da1f-11e9-824d-c814c04d3da1.png)

### VERSIONS USED
- Unity: 2019.1
- ARCore SDK for Unity: ARCore 1.12.0 
- Device manufacturer, model, and O/S: HMD, Nokia 8.1, Android 9.0

**Comment by anyuser** [comment]
I'm experiencing the exact same issues. Tracking seems to work, but the camera image is just green. 

Versions:
- Unity 2019.2.5f1
- ARCore SDK for Unity: 1.12.0
- Nokia 7.1, Android 9

**Comment by MarcelStockerKNN** [comment]
Check mobile build settings, i had a similar issue one time when using ARCore scene, after some researches i discovered that when you build apps for mobile you should disable GPU Skinning* (Project Settings > Player > (Android Settings, expected you change on build settings to android), Other Settings > Rendering). Also check other options, Graphics API should cause error sometimes, so check it out, and try to build again and run the ARCore Scene. Search for Unity mobile guide.

**Comment by anyuser** [comment]
This issue doesn't appear in a build, it's the instant preview that doesn't work. Deactivating GPU Skinning doesn't change anything here.

**Comment by MarcelStockerKNN** [comment]
Check Graphics API, i'm building with OpenGLES3 and OpenGLES2.

**Comment by anyuser** [comment]
I'm also using OpenGLES3 and OpenGLES2.

**Comment by nnonimus** [comment]
Same here guys I am using
- ARCore SDK for Unity: 1.12.0
- Xiaomi redmi k20

**Comment by MarcelStockerKNN** [comment]
This error happens when you build an APK and test on a compatible mobile phone?

**Comment by anyuser** [comment]
No, as mentioned in the original post and my comment, it happens when running instant preview. If I understand right, the point of instant preview is to make it possible to test something without building. In a build there are no problems, everything works as expected.

**Comment by MarcelStockerKNN** [comment]
Sorry if can't help you, me and my buddy decide to test everything building to mobile every time that we want to see the changes. It's a time consuming, but we lost too much more trying to use instant preview and nothing we searched/tried solved the problem. We got no answers on Arcore sdk for unity page on Github, StackOverflow, Unity forum, Reddit. Good luck in find the solution, any progress you achiev share with us

**Comment by anyuser** [comment]
Sure, no problem. It's a pity we have to build every time we change something, it really slows down development. If I find a workaround, I'll mention it here.
